docs/examples/modules/DropdownExample/MultipleSearchSelectionWithAddition.example.vue

```diff
@@ -0,0 +1,43 @@
+<template lang="html">
+  <sui-dropdown
+    multiple
+    fluid
+    :options="skills"
+    placeholder="Skills"
+    search
+    selection
+    addition
+    v-model="current"
+  />
+</template>
+
+<script>
+export default {
+  name: 'SearchSelectionExample',
```

Comment:
Put please, correct name here: `MultipleSearchSelectionWithAddition`

Reply:
Sorry, I actually did that on purpose because both https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/master/docs/examples/modules/DropdownExample/MultipleSelection.example.vue and https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/master/docs/examples/modules/DropdownExample/MultipleSearchSelection.example.vue had it. I've updated those as well just to clean up a bit.